tests/serac_cuda_smoketest.cpp

```diff
@@ -1,9 +1,33 @@
+#include <cuda_runtime.h>
 #include <gtest/gtest.h>
 
 #include <algorithm>
 
 void vector_add(float* out, float* a, float* b, int n);
 
+TEST(cuda_smoketest, cuda_version)
```

Comment:
Another test to tell me when i forgot to load the right CUDA module..... that definitely hasn't happend to me twice today...

Reply:
I think I am going to push this up to BLT because its amazingly helpful